java/libphonenumber/pom.xml

```diff
@@ -47,6 +47,14 @@
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>com.google.protobuf</pattern>
+                  <shadedPattern>libphonenumber.repackaged.com.google.protobuf</shadedPattern>
```

Comment:
Ah, you're right. I lost my thread of thought due to the line wrapping :) Looks good!
I'll merge and patch this within a week at the latest. I want to release it separately from the metadata release this week, as 7.4.0.